go.mod

```diff
@@ -30,20 +30,21 @@ require (
 require (
 	cloud.google.com/go/compute v1.6.1
 	github.com/BurntSushi/toml v1.1.0
-	github.com/aws/aws-sdk-go v1.44.11
+	github.com/aws/aws-sdk-go v1.44.5
 	github.com/beevik/ntp v0.3.0
 	github.com/cenkalti/backoff/v4 v4.1.3
 	github.com/containerd/cgroups v1.0.4-0.20220301195952-2e502f6b9e43
-	github.com/containerd/containerd v1.6.4
+	github.com/containerd/containerd v1.6.3
```

Comment:
since talos keeps the api version of tools shipped with the OS same as the binary version, this needs to be kept at `1.6.4`

Reply:
Thanks, fixed.